src/main/scripts/mlsearch

```diff
@@ -77,26 +111,28 @@ def main(event):
         response_msg = hp.response(str(ex), 500)
         return response_msg
 
+
 if __name__ == "__main__":
     event = {
-        'query': args['query'],
-        'init_idx': args['init_idx'],
-        'count': args['count'],
-        'source': args['source'],
-        'cookies': args['cookies'],
-        'timestamp': args['timestamp'],
-        'y_next_page_token': args['y_next_page_token']
+        "query": args["query"],
+        "init_idx": args["init_idx"],
+        "count": args["count"],
+        "source": args["source"],
+        "cookies": args["cookies"],
+        "timestamp": args["timestamp"],
+        "y_next_page_token": args["y_next_page_token"],
     }
 
-    if args['pwc_user']:
-        event['pwc_user'] = args['pwc_user']
-    if args['pwc_password']:
-        event['pwc_password'] = args['pwc_password']
-    if args['github_acc_token']:
-        event['github_acc_token'] = args['github_acc_token']
-    if args['youtube_dev_key']:
-        event['youtube_developer_key'] = args['youtube_dev_key']
-
+    if args["pwc_user"]:
+        event["pwc_user"] = args["pwc_user"]
+    if args["pwc_password"]:
+        event["pwc_password"] = args["pwc_password"]
+    if args["github_acc_token"]:
+        event["github_acc_token"] = args["github_acc_token"]
+    if args["youtube_dev_key"]:
```

Comment:
Sorry, I should recognize this earlier. Since at the other place we're using `y`, I think it'll be more readable if we change this into `y` too.